<launch>    
    <node pkg="activity_semantics" type="ped_collector" name="ped_collector" output="screen">
        <param name="map_scale_" value="0.5"/>
        <param name="map_pic_path" value="$(find activity_semantics)/launch/color_edge.jpg"/>
    </node>
</launch>


